### PR TITLE
Fixes #12 - Update to Graced's shopping list

### LINKED ISSUE #12: Update gdangelo8's shopping list for a dinner party
Planning to have a few people over

### PR DESCRIPTION
**Pull request by gdangelo8**
This is the fix to issue #12 
@megaptera-helvetiae, could you please review these changes?

### CODE REVIEW

File: lists/gracedShoppingList.Md [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
-bread
-orange juice
-coffee
+cheese 
+bread 
+orange juice 
+coffee 
 chocolate
+milk  
+liver
```
Comment (megaptera-helvetiae) [inline lists/gracedShoppingList.Md:7]: @gdangelo8 are you sure you need liver? I thought you were vegetarian!

File: lists/gracedShoppingList.Md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
-coffee
+cheese 
+bread 
+orange juice 
+coffee 
 chocolate
+milk  
+liver 
+fava beans 
+chianti
```
Comment (megaptera-helvetiae) [inline lists/gracedShoppingList.Md:9]: @gdangelo8 chianti should be written in capital.

File: lists/gracedShoppingList.Md (hunk 1 above)
Comment (gdangelo8) [reply lists/gracedShoppingList.Md:7]: Thank you for the feedback, I corrected this point at this location: https://github.com/pbuttigieg/shoppingList/pull/25/commits/cd060ccff0feaecf0fcd5eb68aa3d45e8e9b2f76

File: lists/gracedShoppingList.Md (hunk 2 above)
Comment (gdangelo8) [reply lists/gracedShoppingList.Md:9]: Thank you for the feedback, I corrected this point at this location: https://github.com/pbuttigieg/shoppingList/pull/25/commits/cd060ccff0feaecf0fcd5eb68aa3d45e8e9b2f76

**Pull request merged**